packages/create-resolve-app/bin/index.js

```diff
@@ -127,22 +127,46 @@ if (unknownOptions && unknownOptions.length) {
   }
 
   const downloadRepo = () =>
-    new Promise(resolve => {
+    new Promise((resolve, reject) => {
       log(chalk.green('Load example'))
       https.get(resolveRepoPath, function(response) {
         response.on('data', function(data) {
           fs.appendFileSync(tmpFilePath, data)
         })
 
         response.on('end', function() {
-          var zip = new AdmZip(tmpFilePath)
-          zip.extractAllTo(`./${appName}`)
-          fs.unlink(tmpFilePath)
-          resolve()
+          try {
+            var zip = new AdmZip(tmpFilePath)
+            zip.extractAllTo(`./${appName}`)
+            fs.unlink(tmpFilePath)
```

Comment:
Is's asynchronous function, and should have then/catch sections, or be awaited
https://nodejs.org/api/fs.html#fs_fs_unlink_path_callback